templates/typescript-agent/Dockerfile

```
@@ -0,0 +1,166 @@
+
+# Stage 1: Full monorepo context for dependency resolution
+FROM node:20-bookworm-slim AS monorepo-base
+
+# This is to remove the notice to update NPM that will break the output from STDOUT
+RUN npm config set update-notifier false
+
+# Install alternative package managers globally
+RUN npm install -g pnpm@latest
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)